compute/instances/troubleshooting/bootscript-eol.mdx

```diff
@@ -0,0 +1,90 @@
+
+1. [Create a snapshot](/compute/instances/how-to/create-a-snapshot/) of the volume using the **unified** snapshot type.
+2. [Create a new block volume](/storage/block/how-to/create-a-volume) from the previous unified snapshot.
+3. [Create and start](/compute/instances/how-to/create-an-instance/) a new Instance using its own boot volume in local boot mode.
+4. [Attach](/storage/block/how-to/attach-a-volume/) the block volume to the new Instance.
+5. Once logged into the Instance, [mount the partition](/storage/block/how-to/mount-and-use-volume/) available in the block volume and access your data.
+6. Delete the old Instance that was using a bootscript once the migration is completed.
+
+#### Option 2: Create a unified snapshot of the volume(s) and export it to S3 to retrieve the data
+
+1. [Create a snapshot](/compute/instances/how-to/create-a-snapshot/) of the volume using the **unified** or **l_ssd** type of snapshot.
+2. [Export](/compute/instances/how-to/snapshot-import-export-feature) the snapshot to an S3 bucket in the same region as the Instance.
+3. Retrieve your data from the Object Storage bucket and reuse it at your convenience.
+4. Delete the old Instance that was using a bootscript once you have recovered your data.
+
+#### Option 3: Create a new Instance using the local boot method and manually copy data from the old Instance to the new one.
+
+[Migrate the data](/dedibox/dedicated-servers/how-to/migrate-server/) of your old Instance manually. You can do this by using FTP, SCP or Rsync, for example.
+
+<Message type="important">
+  Old volumes (prior to 2018) which include QCOW backing chains are currently not supported in the unified snapshot/snapshot transfer logic. For volumes using backing chains, options 1 and 2 are not applicable.
+</Message>
+
+### Instances using the bootscript method and an OS image with UEFI
+
+#### Option 4: Change the boot type of the Instance to local boot
+
+To continue using your current Instance, [change the boot type](/compute/instances/how-to/use-boot-modes/#how-to-use-local-boot) of the Instance to **Local boot**.
+
+1. [Stop](/compute/instances/how-to/power-off-instance/) or [pause](/compute/instances/how-to/use-standby-mode/) the Instance.
+2. [Change the boot type](/compute/instances/how-to/use-boot-modes/#how-to-use-local-boot) to local boot.
+3. Start or resume execution of the Instance.
+4. Check that the Instance is booting correctly with the local boot method and that its services are behaving correctly.
+
+<Message type="tip">
+  If changing the boot type from bootscript to local boot is not successfully achieved via option 4, you can always use option 1, 2 or 3 to migrate your Instance data to a new one.
+</Message>
+
+<Message type="tip">
+  If you are a Stardust user, we recommend that you stop the VM from the VM shell, so that the VM itself is in `stopped_in_place` state. This ensures your Instance will not lose its slot in the case of availability shortages.
```

Comment:
"[pause](/compute/instances/how-to/use-standby-mode/)" is definitely better handling

Reply:
```suggestion
  If you are a **STARDUST1** user, we recommend that you put your VM into [standby mode](/compute/instances/how-to/use-standby-mode/). This ensures your Instance will not lose its slot in the case of availability shortages.
```